src/time/CMakeLists.txt

```diff
@@ -10,7 +10,9 @@ target_include_directories(Time
         INTERFACE interface
 )
 
-target_link_libraries(Time PUBLIC SDL_1_XX)
+target_link_libraries(Time
+    PUBLIC
+        SDL_1_XX)
```

Comment:
Should have been left inlined, it's one library being linked after all.

This reminds me of adding coding guideline to the readme or adapting existing one.